templates/init/databases.sql

```diff
@@ -3,5 +3,14 @@
 
 -- Create {{ db.name }} database.
 CREATE USER "{{ db.get("user", db.name) }}" PASSWORD '{{ db.pass | mandatory }}';
-CREATE DATABASE "{{ db.name }}" WITH OWNER "{{ db.get("user", db.name) }}";
+
+CREATE DATABASE "{{ db.name }}"
+WITH OWNER "{{ db.get("user", db.name) }}"
+{% if 'locale' in db %}
+LOCALE "{{ db.locale }}"
+{% endif %}
+{% if 'template' in db %}
+TEMPLATE "{{ db.template }}"
+{% endif %}
+;
```

Comment:
nit pick : semi colon on new line looks weird :D 
```suggestion
{% endif %};
```

Reply:
I think it's fine for sql - commands are multiline naturally.
And optional parameters are split with mandatory semicolon.
What do you think?

Reply:
oh right I did not consider the optional condition.
your existing way seems better.

Thank you.